Bumps in harmonic : ci_matching_branch/bump_harmonic_sdformat14

azeey opened the pull request:
See https://github.com/gazebo-tooling/release-tools/issues/976

scpeters commented on jammy/debian/changelog:
```diff
@@ -1,71 +0,0 @@
-sdformat13 (13.5.0-1~jammy) jammy; urgency=medium
```
I think there needs to be at least one entry in a changelog for our deb-builders to work:

* https://github.com/gazebo-release/gz-sensors8-release/blob/main/jammy/debian/changelog

azeey replied:
Fixed in https://github.com/gazebo-release/sdformat14-release/pull/1/commits/fdb76cb117183737c9d111e3d0935cf1f65eb669

scpeters commented:
this would be a little easier to review if the `rm -rf focal` was in a separate commit than the other changes, but I can manage as is

scpeters commented on ubuntu/debian/tests/build:
```diff
@@ -32,11 +32,11 @@ cmake_minimum_required(VERSION 3.5)
 project(sdf_test VERSION 1.0.0)
 
-find_package(sdformat13 13.0.0 REQUIRED)
+find_package(sdformat14 13.0.0 REQUIRED)
```
```suggestion
find_package(sdformat14 14.0.0 REQUIRED)
```

azeey replied:
Fixed in https://github.com/gazebo-release/sdformat14-release/pull/1/commits/fdb76cb117183737c9d111e3d0935cf1f65eb669

Commit:
Remove focal

Signed-off-by: Addisu Z. Taddese <addisu@openrobotics.org>

Commit:
Update sdformat13 to sdformat14

Signed-off-by: Addisu Z. Taddese <addisu@openrobotics.org>

Commit:
Address reviewer feedback

Signed-off-by: Addisu Z. Taddese <addisu@openrobotics.org>

azeey commented:
> this would be a little easier to review if the `rm -rf focal` was in a separate commit than the other changes, but I can manage as is

Yeah, sorry about that. I've separated the changes in two commits and addressed your feedback in the third.

iche033 commented on ubuntu/debian/compat:
```diff
@@ -1 +1 @@
-9
+11
```
checking to see if this is intended? Looking at a few other release repos, the debhelper compatibility levels are set to 9.

azeey replied:
I made the change after seeing https://github.com/gazebo-release/sdformat13-release/pull/18.

iche033 replied:
ah ok I see

iche033 replied:
maybe we also need to change this from 13 to 11: https://github.com/gazebo-release/sdformat14-release/blob/ci_matching_branch/bump_harmonic_sdformat14/jammy/debian/compat

azeey replied:
Oh interesting, I think we actually need 13, so I'll update this file and make the jammy one symlink to this. I found https://manpages.ubuntu.com/manpages/jammy/en/man7/debhelper.7.html#compatibility%20levels which states 13 is the recommended level for Jammy. For Focal, https://manpages.ubuntu.com/manpages/focal/en/man7/debhelper.7.html#compatibility%20levels says 12 is recommended and actually says not to use 11.

iche033 replied:
sounds good!

Commit:
Change compat to 13

According to
https://manpages.ubuntu.com/manpages/jammy/en/man7/debhelper.7.html#compatibility%20levels
compat level 13 is recommended.

Signed-off-by: Addisu Z. Taddese <addisu@openrobotics.org>

azeey commented:
Test build: [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=sdformat14-debbuilder&build=1)](https://build.osrfoundation.org/view/ign-harmonic/job/sdformat14-debbuilder/1/)
